cpp/third_party/vcpkg_overlays/ports/snappy/snappy-disable-bmi.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/snappy.cc b/snappy.cc
+index d414718..7b49d2a 100644
+--- a/snappy.cc
++++ b/snappy.cc
+@@ -1014,14 +1014,10 @@ static inline void Report(const char *algorithm, size_t compressed_size,
+ static inline uint32_t ExtractLowBytes(const uint32_t& v, int n) {
+   assert(n >= 0);
+   assert(n <= 4);
+-#if SNAPPY_HAVE_BMI2
```

Comment:
Why do we need this patch? Is the `SNAPPY_HAVE_BMI2` not working properly on Mac? It seems sensible to use the fancy instruction whenever we can.

Reply:
Let me try to remove it again.